Nightwatch.Tests/Resources/Shell.fs

```diff
@@ -8,25 +9,29 @@ open Xunit
 open Nightwatch.Core.Process
 open Nightwatch.Resources
 
-let private getChecker controller processName =
+let private getChecker controller command =
     let factory = Shell.factory controller
 
-    let param = [| "cmd", processName |] |> Map.ofArray
+    let param = [| "cmd", command |] |> Map.ofArray
     factory.create.Invoke param
 
 [<Fact>]
 let ``Shell Resource starts a process``() =
     let processName = "any.exe"
+    let args = [|"-a"; "--b"; "test"|]
+    let command = processName + " " + String.Join(' ', args)
 
     let mutable startedCommand = None
     let mutable startedArgs = None
-    let controller = { execute = fun name _ ->
+    let controller = { execute = fun name args ->
         startedCommand <- Some name
+        startedArgs <- Some args
         Task.FromResult 0 }
-    let checker = getChecker controller processName
+    let checker = getChecker controller command
     task {
         let! result = checker.Invoke()
         Assert.Equal(processName, Option.get startedCommand)
+        Assert.Equal<string []>(args, Option.get startedArgs)
```

Comment:
I think you could write `Assert.Equal<string>` here, and it will compare enumerables instead of actual arrays.